record/record.app.js

```diff
@@ -144,17 +151,30 @@
                                     hasLoaded: true
                                 };
                                 $rootScope.tableModels[i] = model;
+                                // TODO: Throw unhandled error.
+                                // throw e;
                             });
+                            // TODO: Add a .catch for exceptions from this loop
+                            // .catch(e) {
+                            //  ErrorService.catchAll(e);
```

Comment:
Test if catch is needed here and if the error will be caught outside the closure